app/controllers/admin/enterprises_controller.rb

```diff
@@ -178,7 +179,8 @@ def collection
       when :for_order_cycle
         @order_cycle = OrderCycle.find_by(id: params[:order_cycle_id]) if params[:order_cycle_id]
         coordinator = Enterprise.find_by(id: params[:coordinator_id]) if params[:coordinator_id]
-        @order_cycle = OrderCycle.new(coordinator: coordinator) if @order_cycle.nil? && coordinator.present?
+        @order_cycle = OrderCycle.new(coordinator: coordinator) if
+          @order_cycle.nil? && coordinator.present?
```

Comment:
This also got reverted again.
```suggestion
         @order_cycle ||= OrderCycle.new(coordinator: coordinator) if coordinator.present?
```